CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 #### browser
 
 - Building the browser package is now possible on Windows, thanks to more portable scripts.
+- Asynchronous calls that lead to a redirection when restoring a session are now
+  blocking, to prevent any race condition. 
```

Comment:
As a developer using the library, "race condition" doesn't tell me a lot. Maybe list the error that should go away? That way, if you encountered the error you'll know it's been fixed, and if you didn't, you'll know it doesn't apply to you :)

Reply:
Rephrased in https://github.com/inrupt/solid-client-authn-js/pull/1243/commits/f2b2a864f260ba4e10b23a6367749f16e1213c60